app/src/components/Landing/Styled.ts

```diff
@@ -1,3 +1,28 @@
 /** @format */
 
-import 'styled-components/native';
+import styled from 'styled-components/native';
+import { FuturaBoldMedium } from '../Text';
+
+export const SplashLogoBox = styled.View`
+  display: flex;
+  width: 200px;
+  height: 200px;
+  background-color: ${({ theme }) => theme.colors.purple[1]};
+  margin: 112px auto 0;
+  border-radius: 30px;
```

Comment:
let's change this to 20px for the border radius for now, and the margin top to 120px